### DISCUSSION TITLE
cfde-submit Globus errors

### DISCUSSION THREAD

**Discussion by jeremyjyang** [nih-cfde/published-documentation#248] [Q&A] [answered]
After "cfde-submit login" and an apparently successful Globus authentication, this error occurs: 
"Client has no tokens, either call login() or supply tokens to client on init."
See attached. 
Anyone know the answer or some clues?
![cfde-submit_Globus_ERROR](https://user-images.githubusercontent.com/864556/129809707-01405e03-01c0-460f-85c2-8f4bd4513447.png)

**Comment by ACharbonneau** [comment, marked as answer]
I've not seen this error, I've got someone looking into it

**Reply by ACharbonneau** [reply]
Are you running the latest version of the tool? 0.1.11?

If not I think updating will fix it. The update call for pip is at the bottom of the documentation if you need it https://docs.nih-cfde.org/en/latest/cfde-submit/docs/index.html

**Reply by jeremyjyang** [reply]
Aha, I was not. After upgrading from 0.1.9 to 0.1.11 with
python3 -m pip install --upgrade cfde-submit
then I was able to login.
I am authenticated and my tokens have been cached! I feel so much better now.
Thanks.
